test/library/standard/IO/IOWrite.chpl

```diff
@@ -0,0 +1,10 @@
+use IO;
+
+proc PrintIOWrite(const input){
+ write(input);
+ writeln(input);
```

Comment:
this test needs to check that these functions return `true`. You could use `assert` for that or just print out the return value.